config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -27,23 +29,46 @@ presubmits:
         resources:
           requests:
             cpu: 4
+  # this replaces the bootstrap / scenario based job going forward
+  - name: pull-kubernetes-integration
+    always_run: true
+    decorate: true
+    skip_branches:
+    - release-\d+.\d+ # per-release job
     annotations:
+      fork-per-release: "true"
       testgrid-num-failures-to-alert: '10'
       testgrid-alert-stale-results-hours: '24'
       testgrid-create-test-group: 'true'
-  # this replaces the bootstrap / scenario based job going forward
+    path_alias: k8s.io/kubernetes
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191106-09649c7-master
```

Comment:
what keeps this image up to date? does this tie all release branches from 1.17 forward to the same go version, etc?